agent/softwarecontainer-config.in

```diff
@@ -1,27 +1,35 @@
+
+#
+# General SoftwareContainer configuration.
+#
+# Optional values are commented out. All other values are mandatory.
+# The group SoftwareContainer must be present.
+#
+
 [SoftwareContainer]
 # Set number of containers to preload on startup of SoftwareContainer
-preload-count = 1
+# preload-count = @PRELOAD_COUNT@
 
 # Set to 'true' to not shut down containers when SoftwareContainer exits
-keep-containers-alive = false
+# keep-containers-alive = @KEEP_CONTAINERS_ALIVE_VALUE_STRING@
 
 # Set to 'true' to use the SessionBus
-use-session-bus = false
+# use-session-bus = @USE_SESSION_BUS_VALUE_STRING@
 
 # Timeout in seconds before containers are forcefully shut down
-shutdown-timeout = 1
+# shutdown-timeout = @SHUTDOWN_TIMEOUT@
 
 # Location for all sharted host/container mounts
-shared-mounts-dir = @SHARED_MOUNTS_DIR@
+# shared-mounts-dir = @SHARED_MOUNTS_DIR@
 
 # Path to the LXC configuration used with liblxc
 # NOTE: This option is being deprecated
-deprecated-lxc-config-path = @LXC_CONFIG_PATH@
+# deprecated-lxc-config-path = @LXC_CONFIG_PATH@
```

Comment:
No we should remove it. We have an action to do that planned in the backlog. But we need it at the moment so it has to be there. My thought was that this name will make sure the user expects this to change, and it's easier to find as it sticks out. Do you think it's OK to keep this as it is until we actually remove the need?